src/dbnode/client/session.go

```diff
@@ -2034,168 +1993,6 @@ func (s *session) streamBlocksMetadataFromPeers(
 // TODO(r): Delete this once we delete the V1 code path
 type pageToken interface{}
```

Comment:
We can delete this now too, since the page token will always be of the same type now (that is `[]byte`).